commands/misc/spam.js

```diff
@@ -1,4 +1,5 @@
 const { SlashCommandBuilder } = require('discord.js');
+const { getOptionContent } = require('../../node_modules/helpers/@spam/option-content.js');
```

Comment:
`../../` is unnecessary here; Node already knows where the `node_modules` directory is located regardless of your current working path[^1].

[^1]: [Node.js Docs - Modules - #loading from node modules folder](https://nodejs.org/api/modules.html#loading-from-node_modules-folders)

Reply:
Done, check commit [19ac873](https://github.com/virlancristian/shaorma-bot/pull/2/commits/19ac873f40b21e39c12de2ba66ced0494f8bf132)